tests/conftest.py

```diff
@@ -1,17 +1,20 @@
 import getpass
 import os
+import re
 from urllib.parse import urlparse
 
 import psycopg2
 import pytest
-
-from ocdskingfishercolab import create_connection
+import sql
+from IPython import get_ipython
 
 
 # If this fixture becomes too slow, we can setup the database once, and run each test in a transaction.
 @pytest.fixture
 def db():
-    database_url = os.getenv('DATABASE_URL', 'postgresql://{}:@localhost:5432/postgres'.format(getpass.getuser()))
+    # Don't call this DATABASE_URL, as ipython-sql will try and use it, instaed of the created database url
+    database_url = os.getenv('TEST_DATABASE_URL', 'postgresql://{}:@localhost:5432/postgres'.format(getpass.getuser()))
+    created_database_url = re.sub('/[^/]*$', '/ocdskingfishercolab_test', database_url)
     parts = urlparse(database_url)
```

Comment:
Don't use `re` unnecessarily:

```suggestion
    parts = urlparse(database_url)
    created_database_url = parts._replace(path='/ocdskingfishercolab_test').geturl()
```